pagopa/pagopa-editorial-components
# fix: build fix

## Short description
This commit fix the bug with the build that creates folders that are not necessary other than creating the src folder on top of all components. Removed all test file and snapshot from the first 6 component folders, those created the bug with the EC "dist". The main source of the issue, other than those files, was not discovered. Tried also to remove test files from other folders instead of those, but the result was a "no-change" for the bug fixing. Removing the first 6 folders tests file was the solution.

## List of changes proposed in this pull request
List of files removed:

1. src/components/Abstract/Abstract.test.tsx
2. src/components/Accordion/Accordion.test.tsx
3. src/components/BannerLink/BannerLink.test.tsx
4. src/components/Cards/Cards.test.tsx
5. src/components/Download/Download.test.tsx
6. src/components/Editorial/Editorial.test.tsx
7. src/components/Abstract/__snapshots__/Abstract.test.tsx.snap
2. src/components/Accordion/__snapshots__/Accordion.test.tsx.snap
3. src/components/BannerLink/__snapshots__/BannerLink.test.tsx.snap
4. src/components/Cards/__snapshots__/Cards.test.tsx.snap
5. src/components/Download/__snapshots__/Download.test.tsx.snap
6. src/components/Editorial/__snapshots__/Editorial.test.tsx.snap

## Product

## How to test
Run "npm run build" to see that now in "dist" folder the structure is the same as the 2.2.0 version (that's correct)
